### DISCUSSION TITLE
USDC Transfer is stucked from Oasis Emerald to Solana more than an hour.

### DISCUSSION THREAD

**Discussion by RiddlerKLS** [wormhole-foundation/wormhole#954] [Problems / bugs] [answered]
I tried to transfer USDC from emerald network (Oasis EVM) to Solana and transfer is stucked more than an hour.

![image](https://user-images.githubusercontent.com/8031762/156928103-7d693554-d791-497d-a9ad-e74043403593.png)

It's just stucked like above, and when I try to redeem my transaction then

'requested VAA not found in store' 

Error occurs.

![image](https://user-images.githubusercontent.com/8031762/156928152-b1e16f28-5105-4fb4-9207-696345cdc402.png)

Please check issue and let me know if there is anything I can do more than just redeem it. 

Oasis Emerald Transaction is : 

0x242a2596a9e048fbdf5c863fc67473f32237998f92d9e425c185a386742fd688

Thank you.

**Comment by evan-gray** [comment, marked as answer]
Hi @RiddlerKLS. Looks like enough of the guardians suffered outage on Oasis to prevent the network from reaching consensus. They have been alerted and normal operations should resume shortly. Once enough are back online and a recovery is run, your transaction should be able to be completed. 🙂

**Reply by RiddlerKLS** [reply]
Thanks, Just checked Portal bridge and everything went well :)
